apps/wallet-mobile/src/components/QRCodeScanner/QRCodeScanner.tsx

```diff
@@ -1,11 +1,23 @@
-import {BarCodeScanner} from 'expo-barcode-scanner'
+import {BarCodeBounds, BarCodeScanner, BarCodeScannerResult} from 'expo-barcode-scanner'
 import {Camera} from 'expo-camera'
 import * as React from 'react'
-import {StyleSheet} from 'react-native'
+import {StyleSheet, Text, useWindowDimensions, View} from 'react-native'
+import {Path, Svg} from 'react-native-svg'
 
-export const QRCodeScanner = ({onRead}: {onRead: ({data}: {data: string}) => Promise<boolean>}) => {
+export const QRCodeScanner = ({
+  onRead,
+  maskEnabled = true,
```

Comment:
don't start a boolean with true, in that case invert the logic -> `withoutMask`
```suggestion
  withMask,
```